docs/FAQs.md

```diff
@@ -33,4 +42,68 @@ If the issue persists, navigate to the folder's properties and choose the option
 5. In the Advanced Security Settings, select the option `Replace all child object permission entries with inheritable permission entries from this object`.
 6. Apply the changes and confirm.
 
-</details>
+## `Initialize-NatNetwork` issues
```

Comment:
I think if we robustly script the installation of the HNS module, we won't need this section.